libs/langchain-scripts/src/build_v2.ts

```diff
@@ -480,7 +489,7 @@ async function checkTreeShaking(config: LangChainConfig) {
   }
 
   if (failed) {
-    process.exit(1);
+    throw new Error("Tree shaking checks failed.");
```

Comment:
Would confirm we're not catching these errors upstream (I don't think so, but wouldn't be surprised)

Reply:
Doesn't appear so